blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DBasicNumericIO.java

```diff
@@ -0,0 +1,33 @@
+package com.rackspacecloud.blueflood.io.datastax;
```

Comment:
Can we name this class DBasicRollupIO to make it obvious that it operates with BasicRollup? I get confused this class versus DSimpleNumberIO.